[OCPBUGS36313] Align IBM Z parm files (#78298)
openshift/openshift-docs: modules/ibmz-configure-nbde-with-static-ip.adoc

----

> rd.neednet=1 \

Reviewer: ```suggestion
cio_ignore=all,!condev rd.neednet=1 \
```

console=ttysclp0 \
coreos.inst.install_dev=/dev/<block_device> \// <1>
ignition.firstboot ignition.platform.id=metal \
coreos.inst.ignition_url=http://<http_server>/master.ign \// <2>
coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <3>
ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> \
rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000 \// <4>
zfcp.allow_lun_scan=0
----

> <1> For installations on DASD-type disks, add `coreos.inst.install_dev=/dev/dasda`. Omit this value for FCP-type disks.

Reviewer: ```suggestion
<1> For installations on DASD-type disks, add `coreos.inst.install_dev=/dev/dasda`. For installations on FCP-type disks, use coreos.inst.install_dev=/dev/sda or /dev/disk/by-id/scsi-<serial_number>.
```

<2> Specify the location of the Ignition config file. Use `master.ign` or `worker.ign`. Only HTTP and HTTPS protocols are supported.
<3> Specify the location of the `rootfs` artifact for the `kernel` and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
<4> For installations on DASD-type disks, replace with `rd.dasd=0.0.xxxx` to specify the DASD device.
endif::ibm-z-kvm[]
ifdef::ibm-z-kvm[]
[source,terminal]
----

> rd.neednet=1 \

Reviewer: ```suggestion
cio_ignore=all,!condev rd.neednet=1 \
```
